<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="MainStackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>681</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="LoginAndRegistration">
     <widget class="QStackedWidget" name="LoginAndRegister_InnerSW">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>140</y>
        <width>691</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Register">
       <widget class="QLineEdit" name="reg_username_entry">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="reg_username_label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="reg_password_label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="reg_password_entry">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>220</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="reg_email_entry">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>140</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="reg_email_label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="reg_title">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>231</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;CREATE AN ACCOUNT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reg_submit_btn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>310</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="reg_premiumcode_entry">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>260</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="reg_premiumcode_label">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>240</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Premium Code&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>180</y>
          <width>171</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Nothing here</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Login">
       <widget class="QLabel" name="log_password_label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="log_title">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>231</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;LOGIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="log_username_entry">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="log_password_entry">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>220</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="log_username_label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="log_submit_btn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>320</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget"/>
     </widget>
     <widget class="QPushButton" name="Login_or_register_selector">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Login_or_register_selector_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Register An Account</string>
      </property>
     </widget>
     <widget class="QLabel" name="server_feeback_label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Home">
     <widget class="QStackedWidget" name="Home_InnerSW">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>661</width>
        <height>671</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Chats_List">
       <widget class="QPushButton" name="newchat_btn">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>130</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>New Chat</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="menu_contacts_scroll_area">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>281</width>
          <height>361</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>281</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="contacts_scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>279</width>
           <height>359</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </widget>
       <widget class="QPushButton" name="refresh_btn">
        <property name="geometry">
         <rect>
          <x>304</x>
          <y>130</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh Chats</string>
        </property>
       </widget>
       <widget class="QLabel" name="menu_page_title">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>50</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Contacts&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="broadcast_frame">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>530</y>
          <width>441</width>
          <height>131</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="broadcast_history">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>391</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="broadcast_entry">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>421</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Chat">
       <widget class="QScrollArea" name="chat_history_scroll_area">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>461</width>
          <height>361</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="chat_history_scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>459</width>
           <height>359</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="enter_message_entry">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>450</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Chat_back_btn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Chat_send_btn">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>450</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLabel" name="loading_chat_history_label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Loading</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Search_Contact">
       <widget class="QScrollArea" name="search_results_scroll_area">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>180</y>
          <width>281</width>
          <height>271</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>281</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="search_results_scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>279</width>
           <height>269</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </widget>
       <widget class="QLineEdit" name="search_account_entry">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>150</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="search_account_label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Search Account</string>
        </property>
       </widget>
       <widget class="QPushButton" name="search_account_submit_btn">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>150</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="search_account_back_btn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="menu_main_title">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:700; text-decoration: underline;&quot;&gt;UOD Messenger App&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="menu_made_by">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>750</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:700;&quot;&gt;Made by 100715281&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
